data/random-battles/gen9/teams.ts

```diff
@@ -164,6 +164,12 @@ export class RandomTeams {
 	 */
 	moveEnforcementCheckers: {[k: string]: MoveEnforcementChecker};
 
+	/** Used by .getPools() */
+	private poolsCacheKey: any[] | undefined;
```

Comment:
It feels like this can be more specific than `any[]`.

Reply:
`[string, number, boolean, RuleTable] | undefined` seems to be what it's supposed to be?